src/Database/ODBCPdoStatement.php

```diff
@@ -91,4 +93,9 @@ private function quote($param)
         }
         return $param;
     }
+
+    public function setFetchMode($mode, $className = null, ...$params)
+    {
+     return true;   
```

Comment:
@daniilly Please add a comment in code, that we can do nothing with fetch mode using `odbc_` functions.